vkdgen.py

```diff
@@ -494,5 +494,5 @@ def __init__(self, *args, **kwargs):
 		emitversions=".*",
 		#defaultExtensions="defaultExtensions",
 		addExtensions=r".*",
-		removeExtensions = r"VK_KHR_.*_surface$",
+		removeExtensions = r"VK_(KHR_.*_surface|AMD_.*|NV_.*|NVX_.*|GOOGLE_.*|.*_extension_\d+)$",

```

Comment:
Any particular reason why you are excluding proprietary extensions here? That's good information to put in your commit message or in a comment.